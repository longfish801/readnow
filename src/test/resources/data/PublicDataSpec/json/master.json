{
    "pubyears": [
        "2022"
    ],
    "authors": {
        "0001": {
            "id": "0001",
            "name": "方丈貴恵",
            "hiraName": "ほうじょう きえ",
            "reviews": [
                "2022001"
            ]
        },
        "0002": {
            "id": "0002",
            "name": "山沢晴雄",
            "hiraName": "やまざわ はるお",
            "reviews": [
                "2022002"
            ]
        },
        "0003": {
            "id": "0003",
            "name": "戸田和光",
            "hiraName": "とだ かずみつ",
            "reviews": [
                "2022002"
            ]
        }
    },
    "tags": {
        "0001": {
            "id": "0001",
            "name": "特殊設定ミステリ",
            "reviews": [
                "2022001"
            ]
        },
        "0002": {
            "id": "0002",
            "name": "〈竜泉家の一族〉シリーズ",
            "reviews": [
                "2022001"
            ]
        },
        "0003": {
            "id": "0003",
            "name": "アリバイ崩し",
            "reviews": [
                "2022002"
            ]
        }
    },
    "bests": {
        "0001": {
            "id": "0001",
            "name": "私的ベスト2022",
            "reviews": [
                "2022001"
            ]
        }
    }
}