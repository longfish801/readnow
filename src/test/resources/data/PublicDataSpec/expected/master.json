{
    "pubyears": [
        "2022"
    ],
    "authors": {
        "0001": {
            "id": "0001",
            "name": "方丈貴恵",
            "hiraName": "ほうじょう きえ",
            "reviews": [
                "20220101"
            ]
        },
        "0002": {
            "id": "0002",
            "name": "山沢晴雄",
            "hiraName": "やまざわ はるお",
            "reviews": [
                "20220201"
            ]
        },
        "0003": {
            "id": "0003",
            "name": "戸田和光",
            "hiraName": "とだ かずみつ",
            "reviews": [
                "20220201"
            ]
        },
        "0004": {
            "id": "0004",
            "name": "本格ミステリ作家クラブ",
            "hiraName": "ほんかくみすてりさっかくらぶ",
            "reviews": [
                "20220601"
            ]
        }
    },
    "tags": {
        "0001": {
            "id": "0001",
            "name": "特殊設定ミステリ",
            "reviews": [
                "20220101"
            ]
        },
        "0002": {
            "id": "0002",
            "name": "〈竜泉家の一族〉シリーズ",
            "reviews": [
                "20220101"
            ]
        },
        "0003": {
            "id": "0003",
            "name": "アリバイ崩し",
            "reviews": [
                "20220201"
            ]
        },
        "0004": {
            "id": "0004",
            "name": "アンソロジー",
            "reviews": [
                "20220601"
            ]
        }
    },
    "bests": {
        "0001": {
            "id": "0001",
            "name": "私的ベスト2022",
            "reviews": [
                "20220101"
            ]
        }
    }
}